### DISCUSSION TITLE
How to dynamically update System tray menu with progress

### DISCUSSION THREAD

**Discussion by billywatkins** [tauri-apps/tauri#8508] [Q&A] [answered]
I am trying to create a video converter app using Tauri on Mac. I would like to display the progress of converted video files, similar to the example image provided. I have been going through the [documentation](https://tauri.app/v1/guides/features/system-tray/ ), but I could not find any information.

Thank you in advance.

<img width="254" alt="Screenshot 2023-12-31 at 3 15 59 AM" src="https://github.com/tauri-apps/tauri/assets/124872003/9c1ecf9c-1969-43dc-b6fc-3d4d5e1ad500">

**Comment by FabianLars** [comment, marked as answer]
Isn't this section what you're asking for? https://tauri.app/v1/guides/features/system-tray/#updating-context-menu-items
Specifically this part
```rs
let item_handle = app_handle.tray_handle().get_item("id-of-that-item");
item_handle.set_title("Currently Processing: 4/10").expect("Couldn't update tray menu entry"); // Probably don't wanna panic here though
```

**Reply by billywatkins** [reply]
thanks, it worked!
